pkg/deploy/api/types.go

```diff
@@ -441,6 +441,20 @@ type DeploymentConfigRollbackSpec struct {
 	IncludeStrategy bool
 }
 
+// DeploymentRequest is a request to a deployment config for a new deployment.
+type DeploymentRequest struct {
+	unversioned.TypeMeta
+	// Name of the deployment config for requesting a new deployment.
+	Name string
```

Comment:
We use Name in DeploymentRollback already. Is there any need to have this prefixed with DC while it's already a part of a deployment subresource and has clear documentation of what it stands?